tests/chart_tests/test_houston_configmap.py

```diff
@@ -480,6 +480,44 @@ def test_houston_configmap_with_loggingsidecar_enabled_with_resource_overrides()
     assert "vector" in prod_yaml["deployments"]["loggingSidecar"]["image"]
 
 
+def test_houston_configmap_with_loggingsidecar_enabled_with_securityContext_configured():
+    """Validate the houston configmap and its embedded data with
+    loggingSidecar."""
+    sidecar_container_name = "sidecar-log-test"
+    image_name = "quay.io/astronomer/ap-vector:0.22.3"
```

Comment:
We should use an obviously wrong value for the image_name:

```suggestion
    image_name = "quay.io/astronomer/ap-vector:unittest-tag"
```